pdns/dnsproxy.cc

```diff
@@ -100,7 +100,7 @@ bool DNSProxy::completePacket(DNSPacket *r, const DNSName& target,const DNSName&
       ret2 = stubDoResolve(target, QType::AAAA, ips);
 
     if(ret1 != RCode::NoError || ret2 != RCode::NoError) {
-      g_log<<Logger::Error<<"Error resolving for "<<aname<<" ALIAS "<<target;
+      g_log<<Logger::Error<<"Error resolving for "<<aname<<" ALIAS "<<target<<" over TCP";
```

Comment:
I didn't think we actually did a TCP query ourselves, here, so probably needs better wording. Given our entirely distinct handling for whether the client query came via UDP or TCP, logging the difference does seem sensible.